gluetool/utils.py

```diff
@@ -1014,6 +1014,56 @@ def load_json(filepath, logger=None):
         raise GlueError("Unable to load JSON file '{}': {}".format(filepath, str(exc)))
 
 
+def _load_yaml_variables(data, enabled=True, logger=None):
+    """
+    Load all variables from files referenced by a YAML, and return function to render a string
+    as a template using this variables. The files containing variables are mentioned in comments,
+    in a form ``# !include <filepath>`` form.
+
+    :param data: data loaded from a YAML file.
+    :param bool enabled: when set to ``False``, variables are not loaded and a simple no-op
+        function is returned.
+    :param gluetool.log.ContextLogger logger: Logger used for logging.
+    :returns: Function accepting a string and returning a rendered template.
+    """
+
+    logger = logger or Logging.get_logger()
+
+    # Our YAML reader preserves comments, they are accessible via `ca` attribute of the
+    # YAML data structure (which behaves like a dict or list, but it has additional
+    # powers).
+    if not enabled or not hasattr(data, 'ca') or not hasattr(data.ca, 'comment') or len(data.ca.comment) <= 1:
```

Comment:
I think this deserves a debug log message ...

Reply:
I'm not sure, it just means there are no comments, it's not really a bug... But, meh, it's just a debug message.